src/utils/DragAndDrop.js

```diff
@@ -332,13 +335,17 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
+            var ext = Path.extname(item.name).replace(/^\./, "").toLowerCase();
```

Comment:
Here you can do `var ext = FilerUtils.normalizeExtension(Path.extname(item.name), true);` instead of all this code (NOTE: the second arg is `true` so we will exclude the period in this case, since `LanguageManager.getLanguageForExtension()` assumes there isn't one).